README.md

```diff
@@ -91,10 +91,20 @@ See the [Veres One Method spec](https://w3c-ccg.github.io/didm-veres-one/) for
 context.
 
 ```js
+const v1 = require('did-veres-one');
+// instantiate a driver for each DID method you'll be using, pass in options
+// For Veres One, use mode: 'dev' when testing against a local Veres One node
+const veresOneDriver = v1.driver({ mode: 'test', httpsAgent, documentLoader });
+
+const keyDriver = require('did-key-driver').driver({});
+
 const dids = require('did-io');
 
-const v1 = dids.methods.veres({ mode: 'test' });
-// or use mode: 'dev' when testing against a local Veres One node
+dids.use('did:v1', veresOneDriver);
+dids.use('did:key', keyDriver);
```

Comment:
Why the `did:` prefix vs just `v1` and `key`?

Reply:
I'm thinking - to make it super explicit that it's a did method? You're right that it's not required really.

Reply:
Changed it to just use `v1` instead of `did:v1`.